src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -75,20 +80,32 @@ private static void aggregateBlameAuthorInfo(RepoConfiguration config, FileInfo
         String blameResults = getGitBlameResult(config, fileInfo.getPath());
         String[] blameResultLines = blameResults.split("\n");
         Path filePath = Paths.get(fileInfo.getPath());
+        Long sinceDateInMs = config.getSinceDate().getTime();
+        Long untilDateInMs = config.getUntilDate().getTime();
+        int systemRawOffset = TimeZone.getTimeZone(ZoneId.systemDefault()).getRawOffset();
```

Comment:
The user may pass in a `timezone` via `-t`, if it is the case, should we still use `systemDefault`?

Reply:
The `timezone` via `-t` is only used for the generation time of the report, but not affecting to the `sinceDate` and `untilDate` of the report. The `sinceDate` and `untilDate` is generated based on the system's default.

e.g. the purpose of `timezone` is only used here
<img width="426" alt="Screen Shot 2019-07-17 at 4 43 51 PM" src="https://user-images.githubusercontent.com/32864116/61361090-5e39c480-a8b2-11e9-98c4-ee937627d7f5.png">

Reply:
> The `timezone` via `-t` is only used for the generation time of the report, but not affecting to the `sinceDate` and `untilDate` of the report. The `sinceDate` and `untilDate` is generated based on the system's default.

It would be nice to have the timezone considered for all time-related things such as the start time. That can be a separate PR though.